src/lib/converter.ts

```diff
@@ -0,0 +1,434 @@
+   */
+  private setRequestGroups(): MapKeyedWithString<Paw.RequestGroup> {
+    const document = this.apiParser.api
+    const { paths } = document
+
+    const groups: CreateRequestGroupType[] = [...Object.keys(paths)]
+      .map(group.mapToGroup)
+      .filter((item: CreateRequestGroupType) => item !== null)
+      .map(group.mapToCapitalize)
+      .reduce(group.createGroup, [])
+
+    groups.forEach((item: CreateRequestGroupType) => {
+      this.requestGroups[item.group] = this.context.createRequestGroup(
+        item.group,
+      )
+    })
+
+    this.groupedRequest = groups
+      .map((item: CreateRequestGroupType): GroupedRequestType[] =>
+        [...item.paths].map((path) => ({ group: item.group, path })),
+      )
+      .flat()
+    return this.requestGroups
+  }
+
+  /**
+   * @method createRequestMeta
+   * @summary
+   * @param {String} path
+   * @param {String} group
+   * @returns {Object<any>}
+   */
+  private createRequestMeta(path: string, group: string): any {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const operation = document.paths[path] as OpenAPIV3.PathsObject
+
+    const ctx = Object.entries(operation).map(([verb, value]) => {
+      const requestContext = value as OpenAPIV3.OperationObject
+
+      if (!requestContext || requestContext.deprecated) return null
+
+      const method = verb.toUpperCase()
+      const { summary, description } = requestContext
+      let requestBody = null
+      let responses = null
+      let parameters = null
+      let servers = null
+      let security = null
+
+      if (requestContext.requestBody) {
+        requestBody = requestContext.requestBody
+      }
+
+      if (requestContext.parameters) {
+        parameters = requestContext.parameters
+      }
+
+      if (requestContext.responses) {
+        const accepts = Object.entries(requestContext.responses)
+          .map(([key, value]) => {
+            const ctx = value as OpenAPIV3.ResponseObject
+            return ctx.content ? Object.keys(ctx.content) : []
+          })
+          .flat()
+        responses = accepts.length > 0 ? accepts : '*/*'
+      }
+
+      if (requestContext.servers) {
+        servers = requestContext.servers
+      }
+
+      if (requestContext.security) {
+        security = requestContext.security
+      }
+
+      return {
+        path,
+        group,
+        method,
+        summary,
+        description,
+        requestBody,
+        parameters,
+        responses,
+        servers,
+        security,
+      } as any
+    })
+
+    return ctx
+  }
+
+  /**
+   * @method createRequest
+   * @summary
+   *
+   * @param {Object} item
+   * @param {Number} index
+   * @param {Array<any>} array
+   */
+  private createRequest(item: any, index?: number, array?: any[]): void {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const { title } = document.info
+
+    const request = this.context.createRequest(
+      item.summary || item.path,
```

Comment:
Perhaps you could even use `operationId` here for a nicer name:

```suggestion
      item.summary || item.operationId || item.path,
```